rds_to_s3_pipeline/README.md

```diff
@@ -0,0 +1,65 @@
+ETL Pipeline - RDS to S3
+=================================
+
+Overview
+--------
+This folder implements an **ETL (Extract, Transform, Load)** pipeline designed to extract plant data from an RDS database, transform the data into a structured format, and upload it as a Parquet file to an AWS S3 bucket.
```

Comment:
Could you add more context to this by saying from the short term RDS data to the long-term storage in S3 bucket?